gpbft/gpbft.go

```diff
@@ -555,11 +570,11 @@ func (i *instance) tryConverge() error {
 	// in the last round, consider it a candidate.
 	if !i.isCandidate(winner.Chain) && winner.Justification.Vote.Step == PREPARE_PHASE && possibleDecisionLastRound {
 		i.log("⚠️ swaying from %s to %s by CONVERGE", &i.proposal, &winner.Chain)
-		i.candidates = append(i.candidates, winner.Chain)
+		i.addCandidate(winner.Chain)
```

Comment:
of note: The lack of coverage here means we have no converge sway test case.

Reply:
yep working on it